addon/components/base/bs-form/element.js

```diff
@@ -707,23 +707,45 @@ export default FormGroup.extend({
 
     assert('showValidationOn must be a String or an Array', isArray(showValidationOn) || typeOf(showValidationOn) === 'string');
     if (isArray(showValidationOn)) {
-      return showValidationOn;
+      return showValidationOn.map((type) => {
+        return type.toLowerCase();
+      });
     }
 
     if (typeof showValidationOn.toString === 'function') {
-      return [showValidationOn];
+      return [showValidationOn.toLowerCase()];
     }
     return [];
   }),
 
   /**
    * @method showValidationOnHandler
+   * @params {Event} event
    * @private
    */
-  showValidationOnHandler(event) {
-    if (this.get('_showValidationOn').indexOf(event) !== -1) {
-      this.set('showOwnValidation', true);
+  showValidationOnHandler({ target, type }) {
+    let inputGroupClasses = [
+      // Bootstrap 4
+      '.input-group-append',
+      '.input-group-prepend',
+      // Bootstrap 3
+      '.input-group-addon',
+      '.input-group-btn',
+    ];
```

Comment:
What do you think of making this a (public) property of the component, like `ignoreValidationEventsFrom`, so users could override it if required, e.g. a form element with custom controls that do not use any default Bootstrap addon markup? If so, you could make two separate props in the `bs3`/`bs4` child components.

Reply:
Implemented per 177fe7fe6616214cd6c92e7124635d290e57ed1f.